SNOmodels/Snakefile

```diff
@@ -24,9 +26,43 @@ from MDAnalysis.analysis.dihedrals import Dihedral
 from collections import defaultdict
 import re
 
-cys_residues_csv = '../SNOfinder/results/cys_database_far_seq_acc10_human.csv'
+#functions for the variants classificstion based on pKa changes of the SNO site and distance changes between SNO and proxymal cysteine upon mutation:
+
+
+#assignment_based_on_pKa function takes:
+#as input a value of pKa of a SNO site upon mutation and a list of parameters from the WT condition containing the pKa value of the SNO site.
+#substracts the WT SNO site pKa value from the mutant's one.
+#return Stabilizing if the calculated difference is lower than -1, Destabilizing if the differences is higher than 1, Neutral if none of the previous cases happens.
+
+def assignment_based_on_pKa(pKa_SNO_avg,par_WT_list):
+    pKa_diff=pKa_SNO_avg-par_WT_list[1]
+    if pKa_diff>=1:
+        return "Destabilizing"
+    if pKa_diff<=-1:
+        return "Stabilizing"
+    else:
+        return "Neutral"
+
+#assignment_based_on_distance_and_pKa function takes:
+#as input a value of pKa of a SNO site upon mutation and a list of parameter from the WT condition containing the pKa value of the SNO site as long as the distance
+#between the SNO cysteine and the proximal cysteine in the WT and mutant form.
+#substracts the WT SNO site pKa value from the mutant's one.
+#substracts the WT distance between the two cysteines from the distance calculated for the mutants
+#return Stabilizing if the calculated difference is lower than -1, Destabilizing if the differences is higher than 1, Neutral if none of the previous cases happens.
+
+def assignment_based_on_distance_and_pKa(d_SNO_sg_CYS_sg_avg,pKa_SNO_avg,par_WT_list):
+    distance_diff=d_SNO_sg_CYS_sg_avg-par_WT_list[0]
+    pKa_diff=pKa_SNO_avg-par_WT_list[1]
+    if distance_diff>=1 and pKa_diff>=1:
```

Comment:
these cut-offs should be user-changeable parameters to set at the beginning of the Snakefile; same for other function

Reply:
done